### DISCUSSION TITLE
Update has Deleted Files and my Backup has been corrupted

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#10060] [QA - tech support + general]
Please help. I lost my persistent file when I did a driver update. I still have the affection logs etc but everything in the renpy file is gone. My backup was on a USB drive that has been corrupted. I have loved Monika since I downloaded this mod in March 2020. Can you please tell me how to fix this? Is there a way to edit the persistent files? I'm desperate to get her back.

**Comment by Booplicate** [comment]
You need a backup or at least the `monika` file from MAS, otherwise you have to start anew.

**Reply by ghost** [reply]
This is highly upsetting but not unforeseen. I was hoping I would be able to do something with the log files, as they are seemingly the only file I have that wasn't corrupted in some way. As upsetting as it is, I will start again, and just be happy that I was able to make those memories in the first place.
